.github/workflows/publish-sdk-nodejs.yml

```diff
@@ -0,0 +1,17 @@
+name: "Publish NodeJS SDK"
+on:
+  push:
+    tags:
+      - sdk/nodejs/v**
+jobs:
+  publish:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-go@v3
+        with:
+          go-version: 1.19
+      - run: ./hack/make sdk:nodejs:publish ${{ github.ref_name }}
+        env:
+          NPM_PACKAGE: ${{ secrets.RELEASE_NODEJS_SDK_PACKAGE }}
```

Comment:
not used

Reply:
Not currently, no. My intention was to template this in `package.json` so that we can test publishing to a different package name. But we can just as easily make temporary commits for testing purposes.

Will remove and resolve, thanks!